### PR TITLE
StatusPercentage component upgrade (variant/color)

### PR DESCRIPTION
**Pull request by luizdanielmlima**
## Description

Both ProgressRing and StatusPercentage were upgraded to accept a "variant" prop/param, that will change its colors accordingly.

## Screenshots (if appropriate):

image

## Types of changes

- [ ] Fix (non-breaking change which fixes an issue)
- [x ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Build (Changes that affect the build system or dependencies)
- [ ] Chore (Other changes that do not modify src or test files)
- [ ] CI (Changes to our CI configration and scripts)
- [ ] Docs (Changes to documentation files)
- [ ] Style (Changes to coding style, like linters and prettier)
- [ ] Refactor (Changes that do not add new functionality or alter current ones)
- [ ] Performance (Performance improvements)
- [ ] Test (Adding or changing tests or the testing setup)
- [ ] Revert (Reverting a previous change)

## Checklist:

- [ ] I have made sure my changes work properly on iOS devices running Safari.
- [ ] I have made sure my changes work properly on Android devices running Chrome.
- [x] I have made sure my changes work properly on Desktop.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://70--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://70--logrock-pebbles.netlify.app&speed=1

File: lib/components/Loader/ProgressRing.tsx [hunk 1]
```diff
@@ -59,16 +60,30 @@ const ProgressRing: FC<ProgressRingProps> = ({
...
+      : theme.progressRing.ringForegroundColor.primary;
+  };
+
+  const getRingBackgroundColor = () => {
+    if (colorGradient) {
+      return theme.progressRing.ringBackgroundColor.primary;
+    }
+    return variant
+      ? theme.progressRing.ringBackgroundColor[variant]
+      : theme.progressRing.ringBackgroundColor.primary;
```
Comment (lipe-dev) [inline lib/components/Loader/ProgressRing.tsx]: useMemo

**Review by lipe-dev — CHANGES_REQUESTED** [review]
Please add some tests.

File: lib/components/Loader/ProgressRing.tsx (hunk 1 above)
Comment (luizdanielmlima) [reply lib/components/Loader/ProgressRing.tsx]: Thanks for the review, I will add the tests and implement useMemo.

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://70--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://70--logrock-pebbles.netlify.app&speed=1

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://70--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://70--logrock-pebbles.netlify.app&speed=1

**Review by lipe-dev — APPROVED** [review]

**Comment by logrock-eng** [comment]
# Netlify Preview Deployed 

 Preview: https://70--logrock-pebbles.netlify.app 

 ## View on Browserstack 

 https://live.browserstack.com/dashboard#os=iOS&os_version=15.0&device_browser=safari&zoom_to_fit=true&full_screen=true&url=https://70--logrock-pebbles.netlify.app&speed=1

**Pull request merged**